kata-deploy/build-debug.sh

```diff
@@ -0,0 +1,5 @@
+set -ex
```

Comment:
I guess this file was not really meant to be part of the PR @brtknr ? It also has your repo/usename hardwired ;-)

Reply:
Oops, nope this wasn't meant to be rolled in to the commit.